test/testUtils.ts

```diff
@@ -84,4 +84,8 @@ export async function cleanUpWorkspace(): Promise<any> {
 export function setTextEditorOptions(tabSize: number, insertSpaces: boolean): void {
   Configuration.getInstance().tabstop = tabSize;
   Configuration.getInstance().expandtab = insertSpaces;
+  let options = vscode.window.activeTextEditor.options;
+  options.tabSize = tabSize;
```

Comment:
This should be moved once we can really update Code's configuration through API, which relies on #913. So for now we still have to update the cache option in active text editor.